android/src/com/frostwire/android/gui/fragments/BrowsePeerFragment.java

```diff
@@ -648,6 +656,10 @@ public void onFilterComplete(int i) {
         }
     }
 
+    private void hideCheckBoxMenuItem() {
+            checkBoxMenuItem.setVisible(false);
```

Comment:
remember to use only 4 spaces next time for indentation.
I'll get rid of this method since it only has a one line and we need to save up on symbols.